chapter2/tests/t20/terraform/base/ssh.tf

```diff
@@ -0,0 +1,16 @@
+resource "tls_private_key" "wordpress_cluster" {
+  algorithm = "RSA"
+  rsa_bits  = 4096
+}
+
+resource "aws_key_pair" "wordpress_cluster" {
+  key_name   = var.ssh_cluster_name
+  public_key = tls_private_key.wordpress_cluster.public_key_openssh
+}
+
+resource "local_sensitive_file" "ssh_private_key" {
+  content              = tls_private_key.wordpress_cluster.private_key_openssh
+  filename             = "${path.cwd}/assets/private_keys/${var.ssh_cluster_name}.pem"
```

Comment:
I don't see that your gitignore contains ignore for this path

Reply:
> I don't see that your gitignore contains ignore for this path

@kseniyashaydurova 
Done

Reply:
Still don't see it

Reply:
> Still don't see it

@kseniyashaydurova 
Strangely, it didn't get into the commit, added it back